app/assets/stylesheets/publisher.scss

```diff
@@ -13,12 +13,15 @@
       display: none !important;
     }
 
-    #publisher_textarea_wrapper { border: 1px solid $border-grey !important; }
+    #publisher_textarea_wrapper { border: none; }
```

Comment:
Selector `publisher_textarea_wrapper` should be written in lowercase with hyphens